src/libraries/pkg/baseline/packageIndex.json

```diff
@@ -6455,7 +6455,7 @@
         "5.0.0",
         "5.0.1"
       ],
-      "BaselineVersion": "5.0.1",
+      "BaselineVersion": "5.0.0",
```

Comment:
Curious if we want to do this or not.  
This would mean that JSON wouldn't bring the security fix made to STEW.  What's your thinking @GrabYourPitchforks?
On the plus side, it would mean that ASP.NET doesn't see that new version from STEW (though they might be able to intentionally downgrade).  Is it possible to have ASP.NET test with and without the baseline?

Reply:
It doesn't sound right to downgrade the baseline and miss the security fix when STEW is transitively referenced.

Reply:
We discussed with ASP.NET folks over teams, let's undo the change to the baseline: keep it at 5.0.1 here.  They'll make some customizations to their build to avoid consuming the versioned S.T.E.W.

Reply:
```suggestion
      "BaselineVersion": "5.0.1",
```

Reply:
> This would mean that JSON wouldn't bring the security fix made to STEW.

My main concern would be somebody adding a package ref to _System.Text.Json_, then to _S.T.E.W_, and as a result missing the fix because I believe NuGet will pull in the existing transitive package reference.